src/main/java/com/squareup/kotlinpoet/FunSpec.kt

```diff
@@ -342,6 +359,8 @@ class FunSpec private constructor(builder: Builder) {
     internal const val CONSTRUCTOR = "constructor()"
     internal const val GETTER = "get()"
     internal const val SETTER = "set()"
+    internal const val RETURN = "return "
+    @JvmField internal val RETURN_CODE_BLOCK = CodeBlock.of(RETURN)
```

Comment:
Yeah there's a lot of artifacts from since this codebase started from porting JavaPoet.